pom.xml

```diff
@@ -52,6 +52,11 @@
             <version>${commons.net.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
Gatling has moved to jspecify, like most projects. Use it transitively instead of pulling an object extra lib.